parts/linux/cloud-init/artifacts/cse_config.sh

```diff
@@ -537,7 +537,7 @@ configGPUDrivers() {
         mkdir -p /opt/{actions,gpu}
         if [[ "${CONTAINER_RUNTIME}" == "containerd" ]]; then
             ctr image pull $NVIDIA_DRIVER_IMAGE:$NVIDIA_DRIVER_IMAGE_TAG
-            bash -c "$CTR_GPU_INSTALL_CMD $NVIDIA_DRIVER_IMAGE:$NVIDIA_DRIVER_IMAGE_TAG gpuinstall /entrypoint.sh install" 
+            retrycmd_if_failure 5 10 600 bash -c "$CTR_GPU_INSTALL_CMD $NVIDIA_DRIVER_IMAGE:$NVIDIA_DRIVER_IMAGE_TAG gpuinstall /entrypoint.sh install"
```

Comment:
so I guess worst case this could take an hour? :( 

hopefully that isn't often the case lol

Reply:
tbh those numbers are kind of randomly chosen... I think we should tweak them. I'm open to suggestions Unfortunately the issue that I'm trying to fix with this has been pretty... elusive. One retry should be enough (most of the times...)